scss/components/feedback-editor.scss

```diff
@@ -0,0 +1,31 @@
+.DraftEditor-root {
+  color: var(--text-color);
+  width: 100%;
+  font-size: 18px;
+  margin-top: 7px;
+  min-height: 20px;
+  overflow-wrap: break-word;
+}
+
+.link-tooltip {
+  position: fixed;
+  z-index: 2;
+
+  > .link-tooltip__anchor {
+    position: absolute;
+    width: max-content;
+
+    background-color: var(--twitter-link-background-color);
+    color: var(--twitter-link-text-color);
+    fill: var(--twitter-link-text-color);
```

Comment:
Styles generally look good. Could also use some kinda flexbox to get the share icon to be displayed centered horizontally